apps/condo/domains/user/schema/RegisterNewUserService.js

```diff
@@ -154,17 +154,34 @@ const RegisterNewUserService = new GQLCustomSchema('RegisterNewUserService', {
                     const completedAt = new Date().toISOString()
                     await ConfirmPhoneAction.update(context, action.id, { completedAt, sender, dv: 1 })
                 }
-                const sendChannels = [{
-                    to: { phone: userData.phone },
+                const messages = [{
+                    to: { phone: userData.phone, email: userData.email },
```

Comment:
You specify an email as a possible channel for sending a notification, but you do not check if the user has it. This check goes below